content/business-models/source-available.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Source Available"
+date: 2019-06-04
+weight: 1
+---
+
+While copyleft licenses (e.g., [Parity](https://paritylicense.com/), 
+[AGPL](https://en.wikipedia.org/wiki/Affero_General_Public_License)) were 
+originally created to protect intellectual property of a community, they have 
+not served well in all cases to protect communities from possibly being taken 
+advantage of by a larger corporate entity. For example, a copyleft license may 
+allow for use of open source software by a company, and charging for services 
+afforded by that software. While industry might give back to the community by 
+way of contributions, this is not always the case. In these unfortunate cases, 
+the communities get defensive, and the overall dynamic encourages less openness
+and sharing because of that. Specifically, we see from this practice:
+
+ 1. As a defensive mechanism, software becomes proprietary to protect itself.
```

Comment:
Consider rewriting the dependent clause:
```
In order to defensively protect itself, software becomes proprietary.
```

Reply:
I really like the second, done.